packages/eslint-config-appium-ts/index.mjs

```diff
@@ -0,0 +1,231 @@
+       * I'm sure there's a good reason for it). In this case, a non-null assertion is appropriate.
+       * Often a simple `typeof x === 'y'` conditional is sufficient to narrow the type and avoid the
+       * non-null assertion.
+       */
+      '@typescript-eslint/no-non-null-assertion': 'warn',
+      /**
+       * Sometimes we want unused variables to be present in base class method declarations.
+       */
+      '@typescript-eslint/no-unused-vars': 'warn',
+      /**
+       * Allow native `Promise`s. **This overrides `@appium/eslint-config-appium`.**
+       * @remarks Originally, this was so that we could use [bluebird](https://npm.im/bluebird)
+       * everywhere, but this is not strictly necessary.
+       */
+      'promise/no-native': 'off',
+      /**
+       * Allow `async` functions without `await`.  **This overrides `@appium/eslint-config-appium`.**
+       * @remarks Originally, this was to be more clear about the return value of a function, but with
+       * the addition of types, this is no longer necessary. Further, both `return somePromise` and
+       * `return await somePromise` have their own use-cases.
+       */
+      'require-await': 'off',
+
+      /**
+       * Disables the `brace-style` rule.
+       * @remarks Due to the way `prettier` sometimes formats extremely verbose types, sometimes it is necessary
+       * to indent in a way that is not allowed by the default `brace-style` rule.
+       */
+      'brace-style': 'off',
+    }
+  },
+
+  {
+    ...mochaPlugin.configs.flat.recommended,
+    name: 'Test Overrides',
+    files: ['**/test/**', '*.spec.*js', '-specs.*js', '*.spec.ts'],
+    rules: {
+      ...mochaPlugin.configs.flat.recommended.rules,
+      /**
+       * Both `@ts-expect-error` and `@ts-ignore` are allowed to be used with impunity in tests.
+       * @remarks We often test things which explicitly violate types.
+       */
+      '@typescript-eslint/ban-ts-comment': 'off',
+      /**
+       * Allow non-null assertions in tests; do not even warn.
+       * @remarks The idea is that the assertions themselves will be written in such a way that if
+       * the non-null assertion was invalid, the assertion would fail.
+       */
+      '@typescript-eslint/no-non-null-assertion': 'off',
+      'mocha/no-exclusive-tests': 2,
+      'mocha/no-mocha-arrows': 2,
+      'mocha/max-top-level-suites': 'off',
+      'mocha/consistent-spacing-between-blocks': 'off',
+      '@typescript-eslint/no-unused-expressions': 'off',
+      'mocha/no-setup-in-describe': 'off',
+      'mocha/no-exports': 'off',
+      'import/no-named-as-default-member': 'off',
+      'mocha/no-skipped-tests': 'off',
+    },
+  },
+
+  {
+    name: 'Default Ignores',
+    ignores: [
+      ...(fs.existsSync(gitignorePath) ? includeIgnoreFile(gitignorePath).ignores : []),
+      '**/*-d.ts',
+      '**/build/**',
+      '**/*.min.js',
+      '**/coverage/**',
+      '**/node_modules/**',
```

Comment:
This line may not be necessary since [it's one of the default patterns](https://eslint.org/docs/latest/use/configure/ignore#ignoring-files)

Reply:
removed